x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/install_prebuilt_rules_fleet_package.ts

```diff
@@ -95,12 +92,11 @@ export const installPrebuiltRulesFleetPackage = async ({
 
         return body;
       },
-      retryService,
-      utilityName: installPrebuiltRulesFleetPackage.name,
-      retries: MAX_RETRIES,
-      timeout: ATTEMPT_TIMEOUT,
-      log,
-    });
+      {
+        retryCount: MAX_RETRIES,
```

Comment:
Looks much nicer by removing the log service and using the retryService function directly 💯 

What would be the default values for these configs in case we don't provide it directly? Are they configurable at the test config level?

Reply:
>What would be the default values for these configs in case we don't provide it directly? Are they configurable at the test config level?

I believe hardcoding `retryCount` default value (e.g. to `2`) will be lead to misusage: similar to `retry.waitForWithTimeout` takes `timeout` value, `tryWithRetries` require {retryCount: <value>} for each call.

Few points:
- I strongly believed that if `retryCount` is not defined explicitly, `retry.try` is the correct function to use.
- Wrapping too many API calls with `retry.tryWithRetries` may hide instability of the product. By explicitly setting `retryCount` folks take more responsibility to evaluate individually case to case.